src/ILCompiler.Compiler/src/Compiler/Compilation.cs

```diff
@@ -204,9 +205,24 @@ private void AddCompilationRoots()
                 AddCompilationRootsForRuntimeExports(module);
            }
 
+            AddCompilationRootsForBootstrap();
```

Comment:
The same part in .NET Native for UWP is called Startup. I am wondering whether we should call it the same. 

@MichalStrehovsky @davidwrighton Any opinions?